cmd/operator/main.go

```diff
@@ -73,14 +73,16 @@ func main() {
 			"Priority class at which the collector pods are run.")
 		gcmEndpoint = flag.String("cloud-monitoring-endpoint", "",
 			"Override for the Cloud Monitoring endpoint to use for all collectors.")
+		// TODO(maxamin) Flag is unused, remove once GKE is updated to stop using this flag.
 		caSelfSign = flag.Bool("ca-selfsign", true,
 			"Whether to self-sign or have kube-apiserver sign certificate key pair for TLS.")
 		webhookAddr = flag.String("webhook-addr", ":8443",
 			"Address to listen to for incoming kube admission webhook connections.")
 		metricsAddr = flag.String("metrics-addr", ":18080", "Address to emit metrics on.")
 	)
 	flag.Parse()
-
+	// TODO(maxamin) Solves the "declared but not used", remove after GKE is updated.
+	_ = caSelfSign
```

Comment:
Why is this necessary? We'd update the binary and the flags it is deployed with in one change,no? @pintohutch

Reply:
Yep - the GKE component is safe as we'll remove the option there after we tag and do a release here.

Reply:
So it's safe to delete this flag entirely here